### ISSUE TITLE
Add author rights note of community shields

### ISSUE THREAD

**Issue by tonivj5** [voidcosmos/angular-communities#58]
**Is your feature request related to a problem? Please describe.**
Add a note about author rights of community shields.

**Describe the solution you'd like**
Like it's done in [angular logos](https://angularlogos.com/) (see note at header and footer). I think we could include it at the footer.

**Comment by juanri0s** [comment]
I can pick this up!

Does the team just want it copied verbatim?

> "All logos on this site are the property of their respective owners. Please clear any copyright issues with the owners before using the logos."

**Comment by tonivj5** [comment]
Hey @juanri0s, thanks :-)

I think `All logos on this site are the property of their respective owners.` is enough.

> Not sure if we have a UI style doc anywhere, so any details on styling would also help.

We haven't a styling guidelines by the moment... :cry:

Design has changed recently and we haven't strictly a footer now... so, maybe we could put it above search :thinking:
![image](https://user-images.githubusercontent.com/7110786/86511531-6b6cfd80-bdfa-11ea-8ee9-1b7612c01b89.png)

Or in FAQ, not to disrupt too the design, wdyt @NyaGarcia and @Caballerog?

**Comment by Caballerog** [comment]
This could be a question on the FAQ page, right now @NyaGarcia is working on this issue in this https://github.com/voidcosmos/angular-communities/pull/63

**Comment by juanri0s** [comment]
@Caballerog @tonivj5 Did we decide where we want to place this item? If it is the FAQ page then I guess wait until that PR is merged in?

**Comment by tonivj5** [comment]
Yeah, I think we should wait until #59 is merged, then we'll decide where place the author rigths :+1:

Thanks for your involvement @juanri0s  :-)
